### PR TITLE
Fixing #85

### PR DESCRIPTION
**Pull request by atomicpages**
* Adding polyfill for Node 8/9
* Adding Node 8 to Travis

### CODE REVIEW

**Comment by whitecolor** [comment]
@atomicpages some problems with node 10, also node 12 need to be tested too

**Comment by atomicpages** [comment]
Yep, got a little caught up. Fixing Node 10, 11, 12 now

File: src/sync-dir.ts [hunk 1]
```diff
@@ -4,6 +4,17 @@ import { resolve } from 'path'
 import * as fs from 'fs-extra'
 import { getFileHash } from './copy'
 
+const NODE_MAJOR_VERSION = parseInt(
+  (<any>process).versions.node.split('.').shift(),
```
Comment (atomicpages) [inline src/sync-dir.ts]: Not sure how to properly type this

**Comment by whitecolor** [comment]
Thanks, had published it.

**Pull request merged**